CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 1.2.0   | [PR#377](https://github.com/BBC/psammead/pull/377) Add select and text knobs in psammead-caption stories |
```

Comment:
We dont need to bump/changelog for a change that doesnt reach outside of the `packages` dir :)